Lambdas/GHHooks/EventHandler/Handlers/ReleaseReporter.swift

```diff
@@ -182,21 +209,54 @@ struct ReleaseReporter {
     }
 
     func sendToDiscord(description: String) async throws {
-        let fullName = repo.fullName.urlPathEncoded()
-        let image =
-            "https://opengraph.githubassets.com/\(UUID().uuidString)/\(fullName)/releases/tag/\(release.tagName)"
+        let image = await findUseableImageLink()
 
         try await self.sendToDiscord(
             embed: .init(
                 title: "[\(repo.uiName)] Release \(release.tagName)".unicodesPrefix(256),
                 description: description,
                 url: release.htmlUrl,
                 color: .cyan,
-                image: .init(url: .exact(image))
+                image: image.map { .init(url: .exact($0)) }
             )
         )
     }
 
+    func findUseableImageLink() async -> String? {
```

Comment:
Sometimes Discord fails to show an image for a release. I _think_ this should fix it.